recipes/outrigger/meta.yaml

```diff
@@ -0,0 +1,88 @@
+package:
+  name: outrigger
+  version: "1.0.0"
+
+source:
+  fn: outrigger-1.0.0.tar.gz
+  url: https://pypi.python.org/packages/13/b9/02d6db3140448685a9651918e3c2831401983d8123dd2560320283abddcc/outrigger-1.0.0.tar.gz
+  md5: 6b7c5a8ae752f79c49e2e124e609a099
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - outrigger = outrigger:main
+    #
+    # Would create an entry point called outrigger that calls outrigger.main()
+
+    - outrigger = outrigger.commandline:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest >=3.0.0
+    - pandas >=0.17.0
+    - coverage
+    - gffutils >=0.8.7.1
+    - pybedtools
+    - biopython
+    - joblib
+    - pysam
+    - graphlite
+
+  run:
+    - python
+    - pytest >=3.0.0
+    - pandas >=0.17.0
+    - coverage
+    - gffutils >=0.8.7.1
+    - pybedtools
+    - biopython
+    - joblib
+    - pysam
+    - graphlite
+
+test:
+  # Python imports
+  imports:
+    - outrigger
+    - outrigger.index
+    - outrigger.io
+    - outrigger.psi
+    - outrigger.validate
+
+  commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+    - outrigger --help
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  requires:
+    - flake8
```

Comment:
If you are just running import existence tests and commands (and not an actual test suite), you may not need `pytest` and `flake8`.